lib/Mail/Milter/Authentication/Protocol/SMTP.pm

```diff
@@ -419,7 +458,7 @@ sub smtp_command_mailfrom {
             my $envfrom = command_param( $command,10 );
             $smtp->{'mail_from'} = $envfrom;
             $envfrom =~ s/ BODY=8BITMIME$//;
```

Comment:
with this change I expect that BODY=8BITMIME would be added as a new param, do you see any reason to continue to remove it?

Reply:
Nope. Not sure how I missed that before.